addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/ParamSinksUtils.java

```diff
@@ -17,28 +17,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.zaproxy.zap.extension.ascanrules;
+package org.zaproxy.addon.commonlib;
 
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import org.apache.commons.collections.map.ReferenceMap;
+import org.apache.commons.httpclient.URI;
 import org.apache.log4j.Logger;
 import org.parosproxy.paros.db.DatabaseException;
 import org.parosproxy.paros.model.HistoryReference;
 import org.parosproxy.paros.model.Model;
 import org.parosproxy.paros.network.HttpMalformedHeaderException;
 import org.parosproxy.paros.network.HttpMessage;
 
-public class PersistentXssUtils {
+public class ParamSinksUtils {
 
-    private static int uniqueIndex;
-    public static String PXSS_PREFIX = "zApPX";
-    public static String PXSS_POSTFIX = "sS";
-    private static Map<String, UserDataSource> map;
     private static Map<String, HashSet<Integer>> sourceToSinks;
+    private static Map<String, HashSet<Integer>> sourceToSinksHashCodes;
```

Comment:
Map<String,Set<Integer>> should be better.

Reply:
Done :)